BedrockServer.cpp

```diff
@@ -1563,17 +1564,91 @@ void BedrockServer::postPoll(fd_map& fdm, uint64_t& nextActivity) {
             break;
             default:
             {
-                SWARN("Socket in unhandled state: " << s->state);
+                SWARN("Socket in unhandled state: " << socket->state);
             }
             break;
         }
     }
+}
+
+void BedrockServer::postPoll(fd_map& fdm, uint64_t& nextActivity) {
+    auto startBaseClassPostPoll = chrono::steady_clock::now();
+
+    // Let the base class do its thing. We lock around this because we allow worker threads to modify the sockets (by
+    // writing to them, but this can truncate send buffers).
+    {
+        SAUTOLOCK(_socketIDMutex);
+        STCPServer::postPoll(fdm);
+    }
+
+    auto startAccept = chrono::steady_clock::now();
+    // Accept any new connections
+    set<Socket*> sockets = _acceptSockets(true);
```

Comment:
do we need to pass the comparison struct here too?

Reply:
I don't think it matters here, but it might be good just for consistency.

Reply:
I decided not to do it because `_acceptSocket` returns a regular set of sockets, which I could also change, but it ends up seeming weirdly specific.